### PR TITLE
Uniswap support

### PR DESCRIPTION
**Pull request by ligi**
### What was wrong?

no uniswap support

### How was it fixed?

added support for uniswap

#### Cute Animal Picture

Cute animal picture

### CODE REVIEW

**Comment by davesque** [comment]
Maybe we should change the PR base my address list branch?  The diff might be less confusing in that case.  And we could merge it without it going into master.

**Comment by ligi** [comment]
The diff is really just this: https://github.com/esPass/pretix-eth-payment-plugin/pull/95/commits/75f4ee9dcaedbe9e6adbc70a1ae90ca880cbd0b4 - not sure if it should target your branch as it has nothing to do with the address list. Let's just get your branch merged and I target master

File: pretix_eth/templates/pretix_eth/pending.html [hunk 1]
```diff
@@ -36,10 +34,17 @@
         ({% trans "allows you to pay with WalletConnect and MetaMask" %}).
       </li>
 
+      <li>
+        <a target="blank" rel="noreferrer noopener" href="{{ uniswap_url }}">
+          {% trans "Via uniswap" %}
```
Comment (skylarweaver) [inline pretix_eth/templates/pretix_eth/pending.html:39]:
```suggestion
          {% trans "Via Uniswap" %}
```

File: pretix_eth/templates/pretix_eth/pending.html [hunk 2]
```diff
@@ -36,10 +34,17 @@
...
+        </a>
+        ({% trans "allows you to pay with any uniswap supported token. Please do not change the output amount!" %}).
```
Comment (skylarweaver) [inline pretix_eth/templates/pretix_eth/pending.html:41]:
```suggestion
        ({% trans "allows you to pay with any Uniswap supported token. Please do not change the output amount!" %}).
```

File: pretix_eth/templates/pretix_eth/pending.html (hunk 1 above)
Comment (skylarweaver) [reply pretix_eth/templates/pretix_eth/pending.html:39]: Capitalized

**Pull request merged**